modules/react-native-status/desktop/rctstatus.cpp

```diff
@@ -109,7 +110,13 @@ void RCTStatus::startNode(QString configString) {
     int networkId = configJSON["NetworkId"].toInt();
     QString dataDir = configJSON["DataDir"].toString();
 
+#ifndef __APPLE__
     QString networkDir = "./" + dataDir;
+#else
+    QString appFolder = QDir::homePath() + "/Library/StatusIm";
+    QString networkDir = appFolder + dataDir;
+    qDebug()<<"RCTStatus::startNode networkDir: "<<networkDir<<" homePath: "<<QDir::homePath();
+#endif
```

Comment:
It seems that my initial implementation `QString networkDir = "./" + dataDir;` doesn't work for now (or haven't worked good from the start.) Since, original Obj-C sources started [creation of networkDir directory](https://github.com/status-im/status-react/blob/develop/modules/react-native-status/ios/RCTStatus/RCTStatus.m#L193) , probably, appropriate solution will be to do the same on our side to. For example, what if to implement networkDir initialization as following (based on some my previous findings + what you have in current PR for nice directory creation):
```
#include <QStandardPaths>
...
QString networkDir = QStandardPaths::writableLocation(QStandardPaths::AppDataLocation) + "/" + dataDir;
QDir dir(networkDir);
if (!dir.exists()) {
  dir.mkpath(".");
}
```
Above solves directory existence issues, so, it seems, above should be the only place where we create new directories for geth data.

Reply:
P.S. On Linux with packaging of App into AppImage, there is no possibility to create directory with path `"./" + dataDir` as well.

Reply:
@MaxRis, thanks! Fixed by suggestions.